specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-04-01-preview/cosmos-db.json

```diff
@@ -6383,6 +6383,10 @@
           "items": {
             "type": "string"
           }
+        },
+        "disableLocalAuth": {
+          "description": "Opt-out of local authentication and ensure only MSI and AAD can be used exclusively for authentication.",
```

Comment:
Suggestion:
"Opt-out of key-based authentication and ensure only AAD can be used exclusively for authentication."

Reply:
I was trying to keep it in line with existing docs:
https://docs.microsoft.com/en-us/search/?terms=disableLocalAuth